src/MyHomeworks/Homework7/Task1/ExceptionHandling.java

```diff
@@ -0,0 +1,31 @@
+package MyHomeworks.Homework7.Task1;
+
+public class ExceptionHandling {
+    public static void divideArrayElements(int[] array) {
```

Comment:
public static void divideArrayElements(int[] array, int divisor) {
        int[] dividedArray = new int[array.length];

        try {
            for (int i = 0; i < array.length; i++) {
                dividedArray[i] = array[i] / divisor;
            }
            System.out.println("Divided array elements:");
            for (int num : dividedArray) {
                System.out.println(num);
            }
        } catch (ArithmeticException e) {
            System.out.println("Error: Division by zero is not allowed.");
        } finally {
            System.out.println("Finally block executed.");
        }
    }

Reply:
Example of correct implementation